logging/build.gradle

```diff
@@ -1 +1,3 @@
 slf4jLambdaApi()
+
+applyJetBrainsPlugin(/* withGit4idea */ false)
```

Comment:
Just make sure that this plugin doesn't expose any `com.intellij` dependencies as `api` (coz otherwise each module that depends on `logging` will transitively see `com.intellij` stuff, which we don't want to be the case in backend)

Reply:
It seems like non of `com.intellij` packages are available in `backend*` modules